lib/tasks/cleaner.rake

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+namespace :cleaner do
```

Comment:
I wonder if we want this to be a one-time migration instead of a more "permanent" rake task 🤔 Folks who are more experienced with Rails, can you share your thoughts?